### ISSUE TITLE
inverted commas cannot be input 

### ISSUE THREAD

**Issue by jackfrancismurphy** [jackfrancismurphy/newdle-frontend#44]
If the headline has inverted commas, they cannot be input, as the keyboard only has single quotation marks ‘’ instead of '' as is available on keyboard
